### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

**Comment by hanxiao** [comment]
depending on what `index` is, you can do both, e.g. nothing wrong with `da[1:3] = DocumentArray.empty(2)`

**Comment by cristianmtr** [comment]
> depending on what `index` is, you can do both, e.g. nothing wrong with `da[1:3] = DocumentArray.empty(2)`

Sure, but I think the basic assumption is that `index` is an `int`. Indexing by ranges would be a slightly "advanced" topic

File: docs/fundamentals/documentarray/access-elements.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Most of the examples below only show getting Documents for the sake of clarity.
 da = DocumentArray(...)
 
 da[index]
-da[index] = DocumentArray(...)
+da[index] = Document(...)
```
Comment (hanxiao) [inline docs/fundamentals/documentarray/access-elements.md:30]:
```suggestion
da[index] = Document(...)
da[index] = DocumentArray(...)
```

**Review by cristianmtr — COMMENTED** [review]
looks good

**Review by hanxiao — APPROVED** [review]

**Pull request merged**